src/components/search-bar/search-bar.tsx

```diff
@@ -11,21 +11,27 @@ const SearchBar: React.FC<SearchBarProps> = ({ getInputProps }) => {
   const t = useText();
   return (
     <>
+      <label htmlFor="search-header-field" className="hide-visually">
+        {t("searchHeaderInputLabel")}
+      </label>
```

Comment:
Please look into using [`getLabelProps()` provided by Downshift](https://www.downshift-js.com/downshift/#props-used-in-examples).

I would expect this as a minimum provided the necessary link between label and input element instead of hardcoding ids.

Reply:
That's a great suggestion, thanks! We now use downshift's getLabelProps functionality instead :)